docs/nada-lang-types.md

```diff
@@ -0,0 +1,82 @@
+# Data Types
+
+Overview of the primitive and the compound Nada data types.
+
+## Primitive Data Types
+
+| `Public`                | `Secret`                | `Literals`        |
+| ----------------------- | ----------------------- | ----------------- |
+| `PublicInteger`         | `SecretInteger`         | `Integer`         |
+| `PublicUnsignedInteger` | `SecretUnsignedInteger` | `UnsignedInteger` |
+| -                       | `SecretBlob`            | -                 |
```

Comment:
The `SecretBlob` is not a nada lang type, is a client type to do store and retrieve.